### PR TITLE
Remove Xcode 7 warnings

### PR DESCRIPTION
**Pull request by irace**
@brianmichel

### CODE REVIEW

File: Tests/Info.plist [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.tumblr.$(PRODUCT_NAME:rfc1034identifier)</string>
```
Comment (brianmichel) [inline Tests/Info.plist]: was this just something that happened with new Xcode?

**Comment by brianmichel** [comment]
I don't know if it's a problem or not but this seems to be built with Xcode 7, which is GM, but not public yet. Your call to merge and tag, probably a non-issue, but will specifying these values in the project file prohibit people not using Xcode 7?

File: Tests/Info.plist (hunk 1 above)
Comment (irace) [inline Tests/Info.plist]: Ah, yeah I followed the “upgrade” settings prompt. I can roll this back.

**Comment by irace** [comment]
I rolled back the Info.plist changes but don”t want to rollback the project file changes. I'm just going to merge, I don't think anyone is using this and if they are (without Xcode 7) they can just use the previous version.

**Pull request merged**